integration_test/ops_agent_test.go

```diff
@@ -99,7 +99,7 @@ func workDirForPlatform(platform string) string {
 	if gce.IsWindows(platform) {
 		return `C:\work`
 	}
-	return "/root/work"
```

Comment:
RunRemotely runs without sudo (unlike RunScriptRemotely), so it was hitting errors about lacking sudo. Instead of sprinkling sudo everywhere, I just moved this directory to somewhere accessible without sudo.